javascript/utils/utils.js

```diff
@@ -69,3 +69,46 @@ export let canHaveArtifact = (planet) => {
 
 export let canFindArtifact = (planet) => energy(planet) >= 96;
 export let hasArtifact = (planet) => planet.heldArtifactId != null;
+
+export let canDeposit = (planet) => {
+  return planet && isMine(planet) && !planet.heldArtifactId
+}
+
+export let calcBonus = (bonus) => {
+  return bonus - 100
+}
+
+export let myPlanetsWithArtifacts = () => {
+  return Array.from(df.getMyPlanets())
+    .filter(df.isPlanetMineable)
+    .sort((p1, p2) => parseInt(p1.locationId, 16) - parseInt(p2.locationId, 16));
+}
```

Comment:
This naming is actually false advertising and was causing a bug in artifactory. Can you snag the new methods/names from the latest version I made available? I think the name of this became `myPlanetsWithFindable` and I added the correct implementation for `myPlanetsWithArtifacts`